docs/reference/query-dsl/geo-queries.asciidoc

```diff
@@ -19,7 +19,7 @@ The queries in this group are:
 
 <<query-dsl-geo-distance-query,`geo_distance`>> query::
 
-    Finds document with geo-points within the specified distance of a central
+    Finds document with geo-points within the specified distance from a central
```

Comment:
I think this should be 'documents' for consistency. Also, I think 'of' is actually correct here because it's being used with the word 'within'.